src/settings.c

```diff
@@ -169,6 +169,27 @@ void load_settings(char *cmdline_config_path)
                 "Truncating long lines or do word wrap"
         );
 
+        {
+                char *c = option_get_string(
+                        "global",
+                        "ellipsize", "-ellipsize", "",
+                        "Ellipsize truncated lines on the start/middle/end"
+                );
+
+                if (strlen(c) > 0) {
+                        if (strcmp(c, "start") == 0)
+                                settings.ellipsize = start;
+                        else if (strcmp(c, "middle") == 0)
+                                settings.ellipsize = middle;
+                        else if (strcmp(c, "end") == 0)
+                                settings.ellipsize = end;
+                        else
+                                fprintf(stderr,
+                                        "Warning: unknown truncating\n");
```

Comment:
`unknown truncating` Doesn't say much, a better message would be `unknown value for ellipsize`